app/src/main/java/org/oppia/app/topic/TopicActivity.kt

```diff
@@ -45,5 +50,11 @@ class TopicActivity : InjectableAppCompatActivity(), RouteToQuestionPlayerListen
 
   companion object {
     internal const val TAG_CONCEPT_CARD_DIALOG = "CONCEPT_CARD_DIALOG"
+    /** Returns a new [Intent] to route to [TopicActivity] for a specified topic ID. */
```

Comment:
This comment is for line 37 in this file, `routeToTopicPlayFragment ` function, finish this functionality. As mentioned earlier you will need to fix all TODOs corresponding to #135 in current code.

Reply:
Will do that in different PR

Reply:
Resolved in #285